Gemfile

```diff
@@ -2,12 +2,12 @@ source 'https://rubygems.org'
 
 ruby '2.3.1'
 
-gem 'graphql', '~>0.19'
+gem 'graphql'
 gem 'graphiql-rails'
 
 # This is an optional gem. When included, GraphQL will use a parser written in C
 # instead of the Ruby parser shipped with graphql-ruby.
-gem 'graphql-libgraphqlparser', '~> 0.2'
+# gem 'graphql-libgraphqlparser'
```

Comment:
I removed this because it's hard to install and doesn't add too much in this demo